www/shared/styles/imports/variables.css

```diff
@@ -0,0 +1,73 @@
+@import "./custom-medias";
+
+/* ==========================================================================
+   Colors
+   ========================================================================== */
+
+:root {
+    --color-white: #fff;
+}
+
+/* ==========================================================================
+   Typography
+   ========================================================================== */
+
+:root {
+    --font-montserrat: "Montserrat";
+    --font-family: var(--font-montserrat), sans-serif;
+
+    --font-weight-regular: 400;
+    --font-weight-medium: 500;
+}
+
+/* ==========================================================================
+   Z-index
+   ========================================================================== */
+
+:root {
+    --z-index-base: 0;
+    --z-index-header: 2;
+}
+
+/* ==========================================================================
+   Transition / Animations timing
+   ========================================================================== */
+
+:root {
+    --duration-fast: 0.15s;
+    --duration-standard: 0.3s;
+    --duration-long: 0.4s;
+}
+
+/* ==========================================================================
+   Layout vars
+   ========================================================================== */
+
+:root {
+    /* App maximum width
+    ============================================= */
+    --layout-container-width: 140rem;
+
+    /* Spacing around main contents of the website
+    ============================================= */
+    --layout-edge-spacing-x: 5%;
+
+    /* Header
+    ============================================= */
+    --layout-header-height: 9rem;
+
+    /* Home
+    ============================================= */
+    --layout-home-width: 100vw;
```

Comment:
I'm not the biggest fan of this solution because you're connecting a specific variable (home) from it's context (`home.css`) into the layout.

Any reason to do this? 🤔

Reply:
no, I'm not a fan either. 
We already have this on the boilerplate. I can remove this! 😃

Reply:
done